server/public/layout.css

```diff
@@ -1,3 +1,4 @@
+/* stylelint-disable font-family-name-quotes */
```

Comment:
Instead of disabling the rule, it would be better to
- follow the rule
- or disable it in your configuration
I'm not really sure this is even needed anymore